pennylane/circuit_drawer/representation_resolver.py

```diff
@@ -29,9 +29,12 @@ class RepresentationResolver:
         show_variable_names (bool, optional): Show variable names instead of variable values.
     """
 
-    def __init__(self, charset=UnicodeCharSet, show_variable_names=False):
+    def __init__(self, charset=UnicodeCharSet):
         self.charset = charset
-        self.show_variable_names = show_variable_names
+
+        # TODO: re-add the following attribute, once the string representation of VariableRefs
+        # is re-enabled
+        # self.show_variable_names = show_variable_names
```

Comment:
I'm with Josh here, wouldn't it be easier if you just made sure that `show_variable_names` is `False`? That way it's easier to put back on later.